tests/Mapper/ProgrammesDbToDomain/CoreEntityMapperMasterBrandMappingTest.php

```diff
@@ -35,10 +35,72 @@ public function testGetDomainModelSeriesWithSetMasterBrand()
             $expectedMasterBrandDomainEntity
         );
 
-        $this->assertEquals($expectedEntity, $this->getMapper()->getDomainModelForProgramme($dbEntityArray));
+        $this->assertEquals($expectedEntity, $this->getMapper($options)->getDomainModelForProgramme($dbEntityArray));
+    }
+
+    public function masterBrandOnCurrentEntityDataProvider()
+    {
+        return [
+            [[], $this->createMock(MasterBrand::class)], // default value is true
```

Comment:
Could you add array keys describing the tests so that when they fail we know which it is.